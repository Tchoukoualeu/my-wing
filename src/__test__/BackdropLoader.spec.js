import React from "react";
import { render } from '@testing-library/react'
import BackdropLoader from "../components/BackdropLoader";

describe("BackdropLoader component", () => {
  test("Matches the snapshots for BackdropLoader open true", () => {
    const { asFragment } = render(<BackdropLoader open={true} />);

    expect(asFragment(<BackdropLoader open={true} />)).toMatchSnapshot();
  });

  test("Matches the snapshots for BackdropLoader open false", () => {
    const { asFragment } = render(<BackdropLoader open={false} />);

    expect(asFragment(<BackdropLoader open={false} />)).toMatchSnapshot();
  });
});