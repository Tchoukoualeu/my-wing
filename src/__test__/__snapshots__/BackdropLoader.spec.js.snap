// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackdropLoader component Matches the snapshots for BackdropLoader open false 1`] = `
<DocumentFragment>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root makeStyles-backdrop-2"
    style="opacity: 0; visibility: hidden;"
  >
    <div
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 40px; height: 40px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;

exports[`BackdropLoader component Matches the snapshots for BackdropLoader open true 1`] = `
<DocumentFragment>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root makeStyles-backdrop-1"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <div
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 40px; height: 40px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
